numpy/lib/shape_base.py

```diff
@@ -758,9 +758,9 @@ def array_split(ary, indices_or_sections, axis=0):
     >>> np.array_split(x, 3)
         [array([0.,  1.,  2.]), array([3.,  4.,  5.]), array([6.,  7.])]
 
-    >>> x = np.arange(7.0)
-    >>> np.array_split(x, 3)
-        [array([0.,  1.,  2.]), array([3.,  4.]), array([5.,  6.])]
+    >>> c = np.arange(9)
+    >>> np.array_split(c, 4)
+        [array([0, 1, 2]), array([3, 4]), array([5, 6]), array([7, 8])]
```

Comment:
While you're here, these indents are wrong, and should be aligned with the left of the >>>:
```suggestion
    [array([0, 1, 2]), array([3, 4]), array([5, 6]), array([7, 8])]
```
Same for the one above.

Otherwise, looks good now.